### ISSUE TITLE
SE repository status 'Private' (OAM 'Deposited (Not live)')

### ISSUE THREAD

**Issue by mrdsaunders** [tomwrobel/ora_data_model#172]
if there is a record_publication_date -> repository -status `Public`
else if there is a record_review_status -> repository-status `Private (In review)`
else -> repository-status `Private`

In OAM:
repository -status `Public` -> `Live`
repository-status `Private (In review)` -> `In review`
repository-status `Private` -> `Deposited (Not live)`

It would seem more accurate to crosswalk all records without a record_publication_date as 'In review' since they have been deposited but are not live. Does anyone know what the rationale was for AB to write the crosswalk the way he did originally?

**Comment by tomwrobel** [comment]
@mrdsaunders I think the rationale was that Oxris might want to know (or care) about the distinction between 'deposited (but otherwise untouched)' and 'reviewed (and therefore under embargo if not live)'

We can possibly re-create this logic for objects that have a reviewer action associated with them somewhere in their histories but aren't in the access db

**Comment by mrdsaunders** [comment]
Thanks Tom that's helpful. We can discuss at ORA-SE.

**Comment by mrdsaunders** [comment]
Depending on outcome, solution could be ORA side, crosswalk or both.

**Comment by eugeniobarrio** [comment]
Are those repository-status Private -> Deposited (Not live) ever going to be reviewed?

**Comment by tomwrobel** [comment]
@eugeniobarrio they're probably already reviewed, just that no review status was recorded in ORA3 that corresponds to record_review_status in ORA4. We can probably imply that for those. Otherwise, yes.

**Comment by mrdsaunders** [comment]
We could just as easily change the crosswalk to map `has ORA record but no status` to `In Review` instead of `Deposited (Not live)`. Are there any problems anyone can foresee with that?

We can also have a discussion about whether we actually want a status of 'Deposited (Not touched yet)' and how whether we could get S Ltd to map that to a status that doesn't trigger reminders to deposit.

**Comment by mrdsaunders** [comment]
In the short term, can we do as you suggest Tom and imply a status for those currently with no status (in order that they show as 'In review' in SE)? 

In the longer term, we could use this status to distinguish between Deposited (review not started) and Deposited (in review). This could in turn potentially be used to distinguish between embargo end of 31/12/9999 where it is genuinely indefinite vs temporarily entered pending review.

What are the current values for record_review_status?

**Comment by tomwrobel** [comment]
@mrdsaunders the allowed values for Review status are contained within:

https://gitlab.bodleian.ox.ac.uk/ORA4/reviewinterface/blob/master/config/authorities/review_status.yml

If one is not set, which value would you like? 

@jjpartridge @tobypitts I assume the value "Incomplete/In progress" would indicate that an item was under review but that we had no other information?

**Comment by tobypitts** [comment]
@tomwrobel Yes, we've generally been using the "Incomplete/In progress" value when we have begun a review but have not finished it or arrived at the point where we are able to assign another status

**Comment by mrdsaunders** [comment]
Hi Tom. We don't mind what the value is as we don't interpret any specific values, just the existence of one. Use whatever value is appropriate to the the actual condition of the record.

**Comment by tomwrobel** [comment]
@jjpartridge are we happy with Incomplete/In progress?

**Comment by jjpartridge** [comment]
@tomwrobel @mrdsaunders can you confirm for me the scenario that we applying this value to?

has no record publication date
has no review status recorded

Is that it?

Didn't we create "Deposited (not reviewed)" for this purpose

How many objects do we have that do not have a record publication date and/or a review status? Surely it _is_ only the new deposits that have not yet been reviewed...

**Comment by tomwrobel** [comment]
I think the issue is that lots of records did not have their record_review_status set, but they *have* been reviewed. New deposits in ORA4 are not a problem, because the deposited (not reviewed) status should be set for these records

**Comment by jjpartridge** [comment]
Those records will have record publication dates though - correct? And therefore set as 'Public'

**Comment by mrdsaunders** [comment]
@jjpartridge Corect, it is applied to :
has no record publication date
has no review status recorded

**Comment by mrdsaunders** [comment]
@jjpartridge If you filter OAM by status 'Deposited (not live)' you will see objects that are not recent deposits but which have neither record publication date nor record_review_status.

**Comment by jjpartridge** [comment]
@mrdsaunders Are there many?

**Comment by mrdsaunders** [comment]
@jjpartridge there are 325 in the OAM

**Comment by tomwrobel** [comment]
325 starts to look like a migration edge case. Examples?

**Comment by jjpartridge** [comment]
[Publications in OA policy.csv.xlsx](https://github.com/tomwrobel/ora_data_model/files/4184820/Publications.in.OA.policy.csv.xlsx)

**Comment by jjpartridge** [comment]
Here is an example: https://ora.ox.ac.uk/objects/uuid:9cd3f03f-6dd5-41b4-9d27-e5c3d79e4956 - looks happy in ORA4-REVIEW and ORA-SYMP

**Comment by tomwrobel** [comment]
- https://ora.ox.ac.uk/objects/uuid:9cd3f03f-6dd5-41b4-9d27-e5c3d79e4956 

It's not a failure. We have a duplicate here:

https://oxris.ox.ac.uk/viewobject.html?cid=1&id=815179

has two ORA deposited objects:

- uuid_7b03a960-1377-42f6-a85c-db0ec2961478
- uuid_9cd3f03f-6dd5-41b4-9d27-e5c3d79e4956 

One (the one you linked to) is public and live, the other is not. 

Are the other items similarly affected?

**Comment by mrdsaunders** [comment]
@thomas-wrobel yes that is the common scenario. Here is one with four ORA records, one live the others not:
https://oxris.ox.ac.uk/viewobject.html?cid=1&id=1077177
